app/views/orders/index.html.erb

```diff
@@ -1,2 +1,5 @@
-<h1>Orders#index</h1>
-<p>Find me in app/views/orders/index.html.erb</p>
+<h1>Orders</h1>
+
+<% @orders.each do |order| %>
+  <p><%= link_to order.id, order %></p>
```

Comment:
Have you tested this ? not sure the syntax works

Reply:
The page is working but it not showing only the relevant orders yet.